src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -321,6 +321,38 @@ internal override void OnSelectedItemChanged(object oldSelectedItem, object sele
 			UpdateContentPresenter();
 		}
 
+		protected override void OnPointerEntered(PointerRoutedEventArgs e)
+		{
+			base.OnPointerEntered(e);
+			IsPointerOver = true;
```

Comment:
I'm feeling that setting `IsPointerOver` is common to more controls that it needs to be handled somewhere up the inheritance hierarchy

Reply:
@dr1rrb

Reply:
Yes this `IsPointerOver` flag is already maintained by the `UIElement`: https://github.com/unoplatform/uno/blob/81e9ce8aa5efc132d202777c76f900501611d163/src/Uno.UI/UI/Xaml/UIElement.Pointers.cs#L1127.

BTW: The setter of this flag should be private, but `ButtonBase` is still it ...